.gitlab-ci.yml

```diff
@@ -1,40 +1,22 @@
 # B"H
 # This file is a template, and might need editing before it works on your project.
 # Read more about this script on this blog post https://about.gitlab.com/2016/11/30/setting-up-gitlab-ci-for-android-projects/, by Greyson Parrelli
-image: openjdk:8-jdk
-
-variables:
-  ANDROID_COMPILE_SDK: "27"
-  ANDROID_BUILD_TOOLS: "27.0.3"
-  ANDROID_SDK_TOOLS: "26.1.1"
+#image: openjdk:8-jdk
+image: registry.gitlab.com/avimarineinnovatons/android-image:latest
 
 before_script:
-  - apt-get --quiet update --yes
-  - apt-get --quiet install --yes wget tar unzip lib32stdc++6 lib32z1
-  - wget --quiet --output-document=android-sdk.zip https://dl.google.com/android/repository/sdk-tools-linux-${ANDROID_SDK_TOOLS}.zip
-  - unzip -d android-sdk-linux android-sdk.zip
-  - echo y | android-sdk-linux/tools/bin/sdkmanager "platforms;android-${ANDROID_COMPILE_SDK}" >/dev/null
-  - echo y | android-sdk-linux/tools/bin/sdkmanager "platform-tools" >/dev/null
-  - echo y | android-sdk-linux/tools/bin/sdkmanager "build-tools;${ANDROID_BUILD_TOOLS}" >/dev/null
-  - export ANDROID_HOME=$PWD/android-sdk-linux
-  - export PATH=$PATH:$PWD/android-sdk-linux/platform-tools/
-  - chmod +x ./gradlew
-  # temporarily disable checking for EPIPE error and use yes to accept all licenses
-  - set +o pipefail
-  - yes | android-sdk-linux/tools/bin/sdkmanager --licenses
-  - set -o pipefail
-
-  # - mkdir /builds/semm/HebrewDateNotification/licenses
-  # - printf "d56f5187479451eabf01fb78af6dfcb131a6481e\n8933bad161af4178b1185d1a37fbf41ea5269c55" > /builds/semm/HebrewDateNotification/licenses/android-sdk-license
-
+    - cp /opt/keys/google-services.json /builds/AviMarineInnovatons/BoatAngels/app/src/dev
+    - cp /opt/keys/keys.xml /builds/AviMarineInnovatons/BoatAngels/app/src/main/res/values
```

Comment:
Where are those files saved? How are they protected?

Reply:
those files are in the docker container created from image specify in the beginning of the .gitlab-ci.yml